examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3027 @@
+{
```

Comment:
<p>Not sure how easy to do but the <code>f</code> plot should show the full grid so it can easily be visually compared to <code>g</code> . Right now it looks like <code>f</code> is the full gird one and <code>g</code> the one  on a subdomain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>